acceptance/framework/k8s/kubectl.go

```diff
@@ -102,7 +102,9 @@ func KubectlDelete(t *testing.T, options *k8s.KubectlOptions, configPath string)
 // deletes it from the cluster by running 'kubectl delete -k'.
 // If there's an error deleting the file, fail the test.
 func KubectlDeleteK(t *testing.T, options *k8s.KubectlOptions, kustomizeDir string) {
-	_, err := RunKubectlAndGetOutputE(t, options, "delete", "--timeout=60s", "-k", kustomizeDir)
+	// Ignore not found errors because Kubernetes automatically cleans up the kube secrets that we deployed
+	// referencing the ServiceAccount when it is deleted.
```

Comment:
This is just for the multiport test because I've added the Secrets to deploy alongside the ServiceAccounts, I suppose an alternative approach would be to create the secrets through the go-client instead of using Kustomize. Thoughts?